src/lib/config.ts

```diff
@@ -773,6 +773,11 @@ class default_config {
      *
      */
     perfsamples: string = "10000"
+
+    /**
+     * Whether or not to integrate with the TST tab
+     */
+    treestyletabintegration: boolean = false
```

Comment:
I think this should default to true and we should use the `management` permission to check if TST is installed.

Reply:
Maybe, I was more thinking to let the user enable it incase they have TST installed but not in use.